commons/glossario.json

```diff
@@ -1,242 +1,249 @@
-    "Visualizzatore": "utente appartenente ad un'organizzazione abilitato esclusivamente a visualizzare i luoghi dell'organizzazione. Un singolo utente può essere visualizzatore di più organizzazioni distinte.",
-    "verificatori": "toDelete",
-    "visualizzatore": "toDelete"
-  },
-  "W": {
-    "Way of working": "modo di lavorare del team caratterizzato dall'utilizzo di best practices. Esso viene raffinato nel tempo attraverso la manutenzione migliorativa.",
-    "Web application": "applicazione distribuita accessibile in una rete Internet per mezzo di un'architettura client/server.",
-    "way of working": "toDelete",
-    "web application": "toDelete"
-  },
-  "X": {},
-  "Y": {
-    "YAML": "standard di serializzazione dei dati a misura d'uomo per tutti i linguaggi di programmazione."
-  },
-  "Z": {}
+    "A": {
+        "API": "acronimo di Application Programming Interface, si intende un insieme di procedure e funzioni offerte ai programmatori, ottenendo un'astrazione ad alto livello per facilitare lo sviluppo. Le API espongono blocchi di codice delle librerie di cui fanno parte, permettendo il riuso del codice.",
+        "API REST": "API che rispetta i principi dell'architettura software REST (acronimo di Representational State Transfer). REST \u00e8 un sistema di trasmissione dei dati che utilizza HTTP e fa un grande uso delle sue funzioni, ricorrendo ai comandi GET, POST, PUT, PATCH, DELETE e OPTIONS, riuscendo cos\u00ec ad identificare risorse precise nel Web.",
+        "Account": "il complesso dei dati identificativi di un utente, che gli consentono l'accesso all'applicazione mobile oppure alla web application.",
+        "Amministratore": "utente con privilegi avanzati su tutto il sistema di Stalker, esegue operazioni amministrative e gestisce le richieste che provengono da utenti con permessi limitati.",
+        "Analista": "ruolo assunto da uno o pi\u00f9 membri del team di progetto, \u00e8 responsabile delle attivit\u00e0 di analisi e redige lo Studio di Fattibilit\u00e0 e l'Analisi dei Requisiti.",
+        "Angular": "framework open source, sviluppato principalmente da Google, per lo sviluppo di applicazioni web nel linguaggio TypeScript",
+        "Applicazione mobile": "applicazione software dedicata ai dispositivi di tipo mobile, quali smartphone o tablet. ",
+        "Architettura": "processo di definizione di una soluzione strutturata che soddisfa tutti i requisiti tecnici e operativi, ottimizzando al contempo attributi di qualit\u00e0 comuni come prestazioni, sicurezza e gestibilit\u00e0. Implica una serie di decisioni basate su una vasta gamma di fattori e ciascuna di queste decisioni pu\u00f2 avere un impatto notevole sulla qualit\u00e0, sulle prestazioni, sulla manutenibilit\u00e0 e sul successo complessivo dell'applicazione.",
+        "Attivit\u00e0": "parte di un processo che indica l'insieme di compiti da realizzare per realizzare il prodotto finale e soddisfarne i requisiti.",
```

Comment:
bisogna sistemare i caratteri unicode corrotti; onestamente non so a cosa sia dovuto. sei sulla versione più recente di make-glossary?

Reply:
Non è un vero problema, in quanto facendo la build del documento le lettere accentate nel pdf risultano corrette. Quindi può anche rimanere visto che non dà problemi rilevanti.